modules/migration-debugging-velero-admission-webhooks-ibm-appconnect.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+:_content-type: PROCEDURE
+[id="migration-debugging-velero-admission-webhooks-ibm-appconnect_{context}"]
+= Restoring IBM AppConnect resources
+
+If you experience issues when you use Velero to a restore an IBM AppConnect resource that has an admission webhook, you can run the checks in this procedure.
+
+.Procedure
+
+. Check if you have any mutating admission plug-ins of `kind: MutatingWebhookConfiguration` in the cluster:
++
+[source,terminal]
+----
+$ oc get MutatingWebhookConfiguration
```

Comment:
@RichardHoch, I think this is more correctly:

```
oc get mutatingwebhookconfigurations
```

As all lowercase. The uppercase works, but I think it's more of a convenience. The output is similar to:

```
NAME          WEBHOOKS   AGE
machine-api   2          215d
```